src/renderers/dom/client/__tests__/ReactMount-test.js

```diff
@@ -20,9 +20,12 @@ describe('ReactMount', function() {
   var WebComponents = undefined;
 
   try {
-    WebComponents = require('WebComponents');
+    if (!/PhantomJS/.test(window.navigator.userAgent)) {
```

Comment:
This is the wrong check. We aren't using Phantom when we run these with `grunt test --debug`. We shouldn't require it in any case where we're in a real browser (I think we have some other test like this but I can't find it at the moment, perhaps just checking if `jest` is defined would work)